app/controllers/application_controller.rb

```diff
@@ -10,6 +10,13 @@ class ApplicationController < ActionController::Base
   rescue_from GdsApi::InvalidUrl, with: :cacheable_404
   rescue_from RecordNotFound, with: :cacheable_404
 
+  if ENV["BASIC_AUTH_USERNAME"]
```

Comment:
Should this be using `ENV["REQUIRE_BASIC_AUTH"]` instead?

Reply:
I've removed the `REQUIRE_BASIC_AUTH` env var from app.json now, it wasn't set anywhere and is not much use if the username and passwords aren't set.  Thanks